src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1) {
```

Comment:
The selectors `&--1 &__star:nth-child(-n + 1)`, `&--2 &__star:nth-child(-n + 2)` and so on, are inconsistent. You are using `:nth-child` for some and `:nth-of-type` for others. You should stick to one method to maintain consistency. The [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) emphasize consistency in code.